data/src/main/java/utill/DMAPIhandler.kt

```
@@ -0,0 +1,66 @@
+                409 -> ConflictException(message = message)
+
+                500, 501, 502, 503 -> ServerException(message = message)
+
+                else -> OtherHttpException(
+                    message = message,
+                    code = e.code()
+                )
+            }
+        } catch (e: SocketTimeoutException) { // 408 : 요청시간초과
+            throw TimeOutException(message = e.message)
+        } catch (e: UnknownHostException) { // 클라이언트 인터넷 없음 에러
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)